src/components/Weather.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+
+const VITE_WEATHER_API_KEY = import.meta.env.VITE_WEATHER_API_KEY;
+
+function fetchWeather(
+  signal: AbortSignal,
+  setWeather: React.Dispatch<unknown>,
+) {
+  fetch(
+    `https://weerlive.nl/api/weerlive_api_v2.php?key=${VITE_WEATHER_API_KEY}&locatie=52.08718206955104,5.165697854286648`,
+    { signal },
+  )
+    .then((response) =>
+      response.json().then((data) => {
+        setWeather(data);
+      }),
+    )
+    .catch((error) => {
+      if (error.name === 'AbortError') {
+        setWeather(
+          JSON.parse('{"liveweer": [{ "temp": 0, "image": "bewolt"}]}'),
```

Comment:
```suggestion
          JSON.parse('{"liveweer": [{ "temp": 0, "image": "bewolkt"}]}'),
```

Besides, shouldn't this happen for every error? Not just AbortError?